### ISSUE TITLE
owsproxy adapter fails

### ISSUE THREAD

**Issue by cehbrecht** [bird-house/twitcher#69]
In  `owsproxy.includeme` the adapter code for `owsproxy_config` gets a `config` but it does expect a `request`:
https://github.com/bird-house/twitcher/blob/ad623c0ff4759ab1d87191b2e27701f02d064ca3/twitcher/owsproxy.py#L217

@fmigneault How is the adapter meant to work here?

When I replace the adapter code with:
```
def includeme(config):
    # from twitcher.adapter import get_adapter_factory
    # get_adapter_factory(config).owsproxy_config(config)
    owsproxy_defaultconfig(config)
```
... the owsproxy gets correctly initialized.

**Comment by fmigneault** [comment]
@cehbrecht 
`DefaultAdapter.owsproxy_config` calls the same thing :  
https://github.com/bird-house/twitcher/blob/89d8c03aa48276f39495432ec9c151a778d53e9a/twitcher/adapter/default.py#L33-L36

The `get_settings` method allows as input any of pyramid's `Configurator`, `Registry`, `Request` or literal settings dictionary. So the config is resolved to settings in any case. The input name could be updated to reflect a more appropriate behaviour in this specific case.

I'm not sure what exactly is not correctly initialized.

**Comment by cehbrecht** [comment]
fixed by PR #73.
